src/pages/Results/index.js

```diff
@@ -178,6 +176,25 @@ class Results extends Component {
         accessor: 'end',
         Cell: row => <span>{getDiffDate(row.value)}</span>,
       },
+      {
+        Header: 'Favorited',
+        accessor: 'favorited',
+        show: username === '',
```

Comment:
Did you mean to say `username !== ''` here in order to check for a logged in user?

Reply:
right, sorry! I was testing stuff